packages/dapp-svelte-wallet/api/src/wallet.js

```diff
@@ -128,14 +128,20 @@ export function buildRootObject(_vatPowers) {
         const filter = offer =>
           (status === null || offer.status === status) &&
           offer.requestContext &&
-          offer.requestContext.origin === dappOrigin;
+          offer.requestContext.dappOrigin === dappOrigin;
+        const filteredOffers = offers => {
+          return offers.filter(filter).map(({ id, ...v }) => ({
+            ...v,
+            id: id.slice(dappOrigin.length + 1),
```

Comment:
Can we instead store `rawId` in `lib-wallet.js` line 812 and use that here instead rather than breaking up `id`?